examples/cpp/Schrodinger2D.cpp

```diff
@@ -0,0 +1,156 @@
+#include "mole.h"    
+#include <iostream>    
+#include <cmath>    
+#include <iomanip>    
+#include <vector>    
+#include <fstream>    
+  
+using namespace arma;    
+  
+int main() {    
+  int p = 105;  // Number of time steps for the simulation    
+  double Lxy = 1.0;    
+  int k = 2;  // Order of accuracy    
+  int m = 50; // Grid points in x    
+  int n = 50; // Grid points in y    
+  int nx = 2; // Energy level in x    
+  int ny = 2; // Energy level in y    
+  double dx = Lxy / m; // Step in x    
+  double dy = Lxy / n; // Step in y    
+  double dt = dx/2; // Reduced time step size with additional reduction    

```

Comment:
dt=dx from MATLAB. 
Go carefully line by line and make sure these match up.

Reply:
I did it to reduce the insability in the values i was getting, will change it